README.md

```diff
@@ -104,41 +118,49 @@ val fetchThree: Fetch[(String, String, String)] = (fetchString(1) |@| fetchStrin
 When executing the above fetch, note how the three identities get batched and the data source is only queried once.
```

Comment:
Is it fair to say that we could also use `fetchMany` here to achieve the same result? If so, that'd be worth showing, perhaps before the tuples version.